scripts/configure-installer-tests-cluster-s390x.sh

```diff
@@ -73,9 +73,13 @@ sleep 5
 oc import-image nodejs:latest --from=registry.redhat.io/rhscl/nodejs-12-rhel7 --confirm -n openshift
 sleep 5
 oc annotate istag/nodejs:latest tags=builder -n openshift --overwrite
+oc delete istag java:8 -n openshift
+sleep 5
```

Comment:
@zhengxiaomei123 Why do we have `sleep 5`? Wouldn't this slow down the test execution.

Reply:
@prietyc123 @rnapoles-rh with Sleep 5 to confirm the image is here after importing and to confirm the image tag is deleted . We met error before without sleep 5. The error is not always happening but happening many times.

Reply:
@zhengxiaomei123 I agree with you point but @prietyc123 has a valid point here as well. One way to address both of your concerns could be to use the `oc import-image` command the way it's used here: https://github.com/openshift/odo/blob/f463d096610b9c88d2507d8481d0dd09b47ba650/tests/helper/helper_oc.go#L312-L317

More specifically, use `oc import-image` command with `--request-timeout 5m` flag. That should give us way more time than `sleep 5` would to import the image.

@prietyc123 WDYT?

Reply:
> More specifically, use oc import-image command with --request-timeout 5m flag.

+1, I would really prefer to use `--request-timeout` than `sleep`.

Reply:
@dharmit Thanks very much. I updated the code following your good points. @prietyc123 @mohammedzee1000 @rnapoles-rh Please help review again. Thanks.